dxr/indexers.py

```diff
@@ -12,7 +12,33 @@
 # An unanlyzed string property that points to a value that can be exact- or
 # prefix-matched against and carries start/end bounds for highlighting. Has
 # both a name and a qualname.
-QUALIFIED_NEEDLE = {
+QUALIFIED_FILE_NEEDLE = {
+    'type': 'object',
+    'properties': {
```

Comment:
It might be nice to factor the values of the 'name' and 'qualname' hashes out. Right now they're repeated 4 times.